spaceship.zsh

```diff
@@ -229,6 +232,59 @@ spaceship_git_status() {
   fi
 }
 
+# Uncommitted changes.
+# Check for uncommitted changes in the index.
+spaceship_hg_uncomitted() {
+  if $(hg st | grep -Eq "^(A)"); then
+    echo -n "${SPACESHIP_GIT_UNCOMMITTED}"
```

Comment:
Semantically incorrect. Better to use `$SPACESHIP_HG_UNCOMMITTED` with fallback to `$SPACESHIP_GIT_UNCOMMITTED`